pickle_maker.ipynb

```diff
@@ -0,0 +1 @@
 not os.path.exists(DEFAULT_PICKLE_PATH):\n","  os.makedirs(DEFAULT_PICKLE_PATH)\n","\n","\n","filter_crowds = True\n","remove_empty = True\n","scale_factor = .8\n","train_val_split = .8\n","# bound_box_lower_cutoff = 900 # uses default from constants.py if no value given\n","\n","make_pickle(filter_crowds, remove_empty, scale_factor, train_val_split)\n"],"execution_count":9,"outputs":[{"output_type":"stream","text":["Folder C-True_DE-True_S-80_TVs-80_DSBB-900 exists\n","loading annotations into memory...\n","Done (t=8.52s)\n","creating index...\n","index created!\n","loading annotations into memory...\n","Done (t=0.28s)\n","creating index...\n","index created!\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"O2MRZcYU2TN2","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1616517881218,"user_tz":420,"elapsed":30533321,"user":{"displayName":"Corey Koelewyn","photoUrl":"","userId":"10684872829383928279"}},"outputId":"1455ddcc-cac9-4beb-f5b7-06e6cbf2487c"},"source":["_!python3 train.py --epochs 10 --batch 16 --hourglass 4 --subset 1 --loss \"euclidean_loss\" 2> errors.log > op.log\n","\n","!tail errors.log\n","\n","# To resume, uncomment the following and point it to the subdirectory. Ensure hourglass matches.\n","# !python3 train.py --epochs 100 --batch 12 --hourglass 4 --subset 1.0 --loss \"euclidean_loss\" --resume True --resume-epoch 9 --resume-subdir \"2021-03-13-16h-38m_batchsize_12_hg_4\"\n","\n","# Note about subset: if you decide to run on a subset <1.0, there is no functionality to retrieve the same subset of data if the model is stopped then resumed. For this reason, subset should only be used for quick tests rather than reporting robust/repeatable results. \n","\n","# Note about loss function: Ensure that the loss function matches across training sessions"],"execution_count":null,"outputs":[{"output_type":"stream","text":["WARNING:tensorflow:From /tensorflow-1.15.2/python3.7/keras/backend/tensorflow_backend.py:422: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.\n","\n","WARNING:tensorflow:From /tensorflow-1.15.2/python3.7/keras/callbacks/tensorboard_v1.py:200: The name tf.summary.merge_all is deprecated. Please use tf.compat.v1.summary.merge_all instead.\n","\n","WARNING:tensorflow:From /tensorflow-1.15.2/python3.7/keras/callbacks/tensorboard_v1.py:203: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.\n","\n","2021-03-23 08:17:48.371388: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7\n","2021-03-23 08:17:49.616987: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10\n","WARNING:tensorflow:From /tensorflow-1.15.2/python3.7/keras/callbacks/tensorboard_v1.py:343: The name tf.Summary is deprecated. Please use tf.compat.v1.Summary instead.\n","\n"],"name":"stdout"}]}]}
```

Comment:
<p>This train cell should probs be removed. The <code>--pickle</code>  flag for training should be moved to <code>colab-train.ipynb</code>  so that people know how to use it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/robertklee/COCO-Human-Pose/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>